omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -5,577 +5,727 @@
-                disable_fields("duration-on-art");
+        .encounter-details {
+        margin-top: 68px;
+        margin-left: 120px;
+        width: 70%;
+        }
 
-                var durationOnART;
-                var durationOnCurrentRegimen;
+        .submitButton {
+        width: 50%;
+        }
+
+        form select[disabled], .form select[disabled] {
+        background: #EEE;
+        border: #1dacbd 1px solid;
+        color: #060606;
+        font-style: normal;
+        }
+
+        form select, .form select {
+        min-width: 100%;
+        max-width: 100%;
+        }
+
+        form input {
+        min-width: 100%;
+        max-width: 100%;
+        }
+
+        .add-more-link {
+        color: dodgerblue;
+        }
+
+        .remove-more-link {
+        color: darkred;
+        }
+
+
+    </style>
 
-                if (artStartDate !== "") {
-                    durationOnART = periodBetweenDates(new Date(artStartDate), new Date(jq("#encounterDate").find("input[type=hidden]").val()));
-                    jq("#duration-on-art input").val(durationOnART);
-                }
 
-                if (currentRegimenStartDate !== "") {
-                    durationOnCurrentRegimen = periodBetweenDates(new Date(currentRegimenStartDate), new Date(jq("#encounterDate").find("input[type=hidden]").val()));
-                    jq("#duration-on-current-regimen input").val(durationOnCurrentRegimen);
+    <script type="text/javascript">
+        if (jQuery) {
+        jq(document).ready(function () {
+        $allRows = jq(this).find('.multi-drug');
+        $allRows.hide();
+        jq('#1-removeEntry').remove();
+        jq('#10-addEntry').remove();
+        jq('#1-toggleContainer').show();
+
+        jq('#11-removeEntry').remove();
+        jq('#20-addEntry').remove();
+        jq('#11-toggleContainer').show();
+        });
+
+
+        jq(document).ready(function () {
+        jq('.addEntry').on("click", function () {
+        var correctedAddButtonId = parseFloat(this.id) + 1;
+        var contentAddId = "#" + correctedAddButtonId + "-toggleContainer";
+        jq(contentAddId).toggle(true);
+        jq('#' + this.id).toggle(false);
+        jq('#' + parseFloat(this.id) + '-removeEntry').toggle(true);
+        return false;
+        });
+        });
+
+        jq(document).ready(function () {
+        jq('.removeEntry').on("click", function () {
+        var correctedRemoveButtonId = parseFloat(this.id) - 1;
+        var contentAddId = "#" + parseFloat(this.id) + "-toggleContainer";
+        jq(contentAddId).toggle(false);
+        jq(':input:not(:button)', contentAddId).val([]);
+        jq('#' + correctedRemoveButtonId + '-addEntry').toggle(true);
+        jq('#' + correctedRemoveButtonId + '-removeEntry').toggle(true);
+
+        return false;
+        });
+
+        });
+        }
+    </script>
+    <script type="text/javascript">
+
+        <ifMode mode="ENTER,EDIT" include="false">
```

Comment:
@slubwama you seem to have copied and pasted from the top

Reply:
@ssmusoke  this was part of this form. I only brought in the POC part

Reply:
done